Templates/AppSource App/.AL-Go/localDevEnv.ps1

```diff
@@ -15,35 +15,6 @@ Param(
 $ErrorActionPreference = "stop"
 Set-StrictMode -Version 2.0
 
-$pshost = Get-Host
```

Comment:
There a lot of common functionality in these scripts for AppSource App and PTE templates. Is there a way to extract such common functionality so that it's not duplicated?

Reply:
Majority of the template repositories are the same - we could have a common and then have all things, which are the same reside in common and only special things be in AppSource App or PTE folders - meaning that when deploying, we would grab Common + PTE to become PTE and Common + AppSource App to become AppSource App.
Make sense?